packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -425,21 +425,33 @@ class _AuthenticatorState extends State<Authenticator> {
     if (cognitoPlugin == null) {
       return const ['auth.plugins.Auth.Default'];
     }
+    if (cognitoPlugin.usernameAttributes == null) {
+      missingValues.add('\n - usernameAttributes');
+    }
     if (cognitoPlugin.signupAttributes == null) {
-      missingValues.add('auth.plugins.Auth.Default.signUpAttributes');
+      missingValues.add('\n - signUpAttributes');
     }
     if (cognitoPlugin.passwordProtectionSettings == null) {
-      missingValues.add('auth.plugins.Auth.Default.passwordProtectionSettings');
+      missingValues.add('\n - passwordProtectionSettings');
     }
     return missingValues;
   }
 
   @override
   Widget build(BuildContext context) {
     if (_configInitialized && _missingConfigValues.isNotEmpty) {
-      throw StateError('''Encountered problem(s) building the Authenticator.
-          Your amplifyconfiguration.dart file is missing required values: ${_missingConfigValues.join('\n')}). 
-          You should correct your amplifyconfiguration.dart file and restart your app.''');
+      throw StateError(
```

Comment:
Can also consider using `FlutterError` which handles this for you. See the inherited widgets for examples.

Reply:
Thanks. I will look into FlutterError and update this.